build-hugo.sh

```
@@ -0,0 +1,31 @@
+HUGO_VERSION=v0.111.3
+
+# Ensure Go environment variables are set
+export PATH=$PATH:/usr/local/go/bin
+export GOPATH=$HOME/go
+export GOBIN=$GOPATH/bin
+mkdir -p $GOPATH/bin
+
+# Install Hugo Extended
+CGO_ENABLED=1 go install -tags extended github.com/gohugoio/hugo@$HUGO_VERSION
+
+echo 'export PATH=$PATH:$GOBIN' >> ~/.bashrc
```

Shellcheck (0.10.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)